src/components/home/MarketingBlock.svelte

```diff
@@ -8,38 +8,42 @@ $: $t = 1;
 </script>
 
 <style>
-.marketing-block {
-  padding: var(--space-2x);
-    background: linear-gradient(to top, white, transparent), url('bg.jpg');
-    background-position: top right -100px;
-  background-repeat: no-repeat;
-}
-
-.marketing-block__inner-container {
-  width: var(--space-72x);
-}
-
-.marketing-block__title {
-  margin:0;
-  margin-bottom: var(--space-4x);
-  line-height:1.2;
-  font-size: var(--text-08);
-}
-
-.marketing-block__description {
-  margin-bottom: var(--space-4x);
-}
-
-.marketing-block__call-to-action {
-  margin-top: var(--space-2x);
-  margin-bottom: var(--space-4x);
-}
-
-.marketing-block__ask-us {
-  font-weight: 200;
-  font-style: italic;
-}
+  .marketing-block {
+    padding: var(--space-2x);
+      background: linear-gradient(to top, white, transparent), url('bg.jpg');
```

Comment:
confused by this – looks like this has the same indent rule as the original but it's just got an additional indent